includes/common-customize-dropdown-menu.md

```diff
@@ -0,0 +1,3 @@
+On desktops and iOS devices, the drop-down menu is the [Popover](/concepts/05%20Widgets/Popover/00%20Overview.md '/Documentation/Guide/UI_Components/Popover/Overview/') UI component; on other devices, it is the [Popup](/concepts/05%20Widgets/Popup/00%20Overview.md '/Documentation/Guide/UI_Components/Popup/Overview/') UI component. To use the Popup on all devices, assign **false** to the [usePopover](/api-reference/10%20UI%20Widgets/dx{WidgetName}/1%20Configuration/usePopover.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#usePopover') property.
```

Comment:
Afaik, only Lookup editor uses the Popover component by default. Other drop-down editors use Popup unless usePopover is specified